Read term dates from manual source in ep-data repo

ondenman opened the pull request:
The end date for term 44 was incorrect, causing problems with the data.

ondenman committed:
Correct term 44 end date

End of term 44 set to 9th May 2016. Source: http://www.ipu.org/parline-e/reports/2015_E.htm

tmtmtmtm reviewed (changes requested):
You really need to include a source for this information in the commit message.

However, maintaining this list in multiple places is also providing problematic, so I'd say this should be entirely replaced with looking it up using everypolitician-ruby:

`EveryPolitician::Index.new.country('Australia').legislature('Representatives').legislative_periods`

ondenman committed:
Reads term dates from terms.csv

Instead of using hardcoded term dates, scraper reads from terms.csv in the ep-data repo (/Australia/Representatives/sources/manual/), making it easier to maintain the list of dates; one source as opposed to hardcoded in multiple scrapers.

ondenman commented:
Term dates now read from manual file in ep-data repo.

tmtmtmtm commented on Gemfile:
```diff
@@ -14,3 +14,4 @@ gem "nokogiri"
 gem "open-uri-cached"
 gem "fuzzy_match"
 gem 'wikidata-client', '~> 0.0.7', require: 'wikidata'
+gem 'everypolitician'
```
I don't think you need to include this any more, do you?

tmtmtmtm reviewed (changes requested):
You've removed the Gemfile, but not the Gemfile.lock

tmtmtmtm reviewed (changes requested):
You still have what appear to be needless changes to `Gemfile.lock`

Rather than adding https://github.com/everypolitician-scrapers/australia-openaustralia/pull/4/commits/aafb75a43bdf96a4ba1e4e61be8f06550b064631 to revert https://github.com/everypolitician-scrapers/australia-openaustralia/pull/4/commits/c722a04d9cd683f11679d304f2954c128cf86c7d, I'd suggest you simply drop both commits.

ondenman commented:
@tmtmtmtm I've rebased and removed both of the gemfile commits.
